doc/plugin_server_notifier_k8sbundle.md

```diff
@@ -14,6 +14,7 @@ The plugin accepts the following configuration options:
 | config_map            | The name of the ConfigMap                   | `spire-bundle`  |
 | config_map_key        | The key within the ConfigMap for the bundle | `bundle.crt`    |
 | kube_config_file_path | The path on disk to the kubeconfig containing configuration to enable interaction with the Kubernetes API server. If unset, it is assumed the notifier is in-cluster and in-cluster credentials will be used. | |
+| label                 | If set, rotate the CA Bundle in validating and mutating webhooks with this label set to `true`. | |
```

Comment:
I had that originally, I moved it to regular `label` because there are other kubernetes types that have cabundles, for example `APIService` is one we use. I tried to write this to be extensible enough to add in these additional types as needed.

Reply:
I don't know. I could see somebody being surprised to upgrade SPIRE and suddenly this plugin start watching and updating other resource types without operator buy-in.

@evan2645, curious to know your thoughts here.

Reply:
We definitely don't want to surprise anyone. This new setting is disabled by default so if they use the same config when they upgrade, there won't be any change in behavior.

Reply:
Sure. But I'm talking about a future change after this one where the plugin decides it wants to update additional K8S objects using the label. Operators who have already opted in to have the plugin manage webhooks would not have to explicitly opt in to get the new behavior for the new K8S objects. Furthermore, in order to _opt out_ of having those new objects managed, operators would have to define new labels, since the configurable is "all-or-nothing" for all object types with that label.

Reply:
oh yea ok, that makes sense. cert-manager has them all lumped together so i was copying that pattern. We can have the types broken out into different configs here to avoid any surprises.